Core/Core/SwiftUIViews/SelectableTextViewWrapper.swift

```diff
@@ -0,0 +1,66 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct SelectableTextViewWrapper: UIViewRepresentable {
+    // MARK: - Properties
+    private let textType: TextType
+    private let font: UIFont.Name
+    private let textColor: Color
+
+    // MARK: - Init
+    public init(
+        textType: TextType,
```

Comment:
IMHO this init could be replaced with one expecting a `text: String` and another expecting `attributedText: AttributedString`. Those could setup the `textType` property, and `TextType` could be private.
But if you rather prefer it this way that's also ok.